drivers/dac/dac_mcux_lpdac.c

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2023, NXP
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Why is this BSD licensed? If possible, please use Apache 2.0 and the same header style as everywhere else.